site/src/pages/RoadmapPage/Planner.tsx

```diff
@@ -309,6 +307,9 @@ const Planner: FC = () => {
         placeholderName={'Year ' + (data.length + 1)}
         placeholderYear={data.length === 0 ? new Date().getFullYear() : data[data.length - 1].startYear + 1}
       />
+      <ImportTranscriptPopup />
+      <br />
+      <br />
```

Comment:
What are the 2 `<br />`s for?

Reply:
was messing with spacing but it doesn't do anything if there's not enough content on the left anyways so gonna remove this